test/support/utils.h

```diff
@@ -574,43 +574,32 @@ class AssocOp
 template <typename T>
 struct Matrix2x2
 {
-    T a[2][2];
-    Matrix2x2() : a{{1, 0}, {0, 1}} {}
-    Matrix2x2(T x, T y) : a{{0, x}, {x, y}} {}
-    //Explicit definition of a copy constructor and assignment operator to avoid an error for some compilers
-#if !_PSTL_ICL_19_VC14_VC141_TEST_SCAN_RELEASE_BROKEN
```

Comment:
Does the new code work for _PSTL_ICL_19_VC14_VC141_TEST_SCAN_RELEASE configuration?

Reply:
I'll check. Thanks.

Reply:
I've just checked. The new code works without issues